File: ws/cañarte/u2/WS24Zoo/ZOO/src/main/java/ec/edu/espe/zoo/model/Primate.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Saray Cañarte, codecreators, DCCO - ESPE
 */
public abstract class Primate extends Mammal{

    public Primate(int id, String description, Date dateOnborn, char gender, int numberOfBones, Cage cage) {
        super(id, description, dateOnborn, gender, numberOfBones, cage);
    }

}

File: hw/ccedeno/u2/HW18ZooInPythonJavaScript/ZooInJavaScript/ec.edu.espe.Zoo.model/Primate.js
const Mammal = require('../ec.edu.espe.Zoo.model/Mammal');
class Primate extends Mammal {
  constructor(id, description, dateOnborn, gender, numberOfBones, cage) {
    super(id, description, dateOnborn, gender, numberOfBones, cage);
  }
}

module.exports = Primate;
